tajo-common/src/main/java/org/apache/tajo/exception/ErrorMessages.java

```diff
@@ -72,6 +72,7 @@
     ADD_MESSAGE(UNDEFINED_INDEX_FOR_TABLE, "index ''%s' does not exist", 1);
     ADD_MESSAGE(UNDEFINED_INDEX_FOR_COLUMNS, "index does not exist for '%s' columns of '%s' table", 2);
     ADD_MESSAGE(UNDEFINED_INDEX_NAME, "index name '%s' does not exist", 1);
+    ADD_MESSAGE(UNDEFINED_TABLESPACE_HANDLER, "No tablespace handler for URI scheme '%s'", 1);
```

Comment:
IMHO, kind and plentiful (not detailed) error messages will make users happy. How about adding an warning message like ```Please check your table location uri scheme```?